ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2853,30 +2853,69 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-nutanix-ipi-boot-categories-project
-- as: nutanix-ipi-p2-f7
+- as: nutanix-ipi-boot-categories-project-p3-f28-destructive
   cluster: build01
-  cron: 33 12 1,8,16,24 * *
+  cron: 47 9 8 * *
   steps:
     cluster_profile: nutanix-qe
+    env:
+      E2E_RUN_TAGS: '@amd64 and @nutanix-ipi and @network-ovnkubernetes and not @fips
+        and @destructive'
+      TAG_VERSION: '@4.13'
+      TEST_ADDITIONAL: Network_Observability
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-nutanix-ipi-boot-categories-project
+- as: nutanix-ipi-disconnected-p2-f28
+  cluster: build01
+  cron: 16 5 3 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      E2E_RUN_TAGS: '@amd64 and @nutanix-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips'
+      TAG_VERSION: '@4.13'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-nutanix-ipi-disconnected
+- as: nutanix-ipi-disconnected-p3-f28-destructive
+  cluster: build01
+  cron: 49 22 2 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      E2E_RUN_TAGS: '@amd64 and @nutanix-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips'
+      TAG_VERSION: '@4.13'
+      TEST_ADDITIONAL: Network_Observability
```

Comment:
I guess we do not need the TEST_ADDITIONAL setting here.

Reply:
So only leave it in nutanix-ipi-boot-categories-project-p3-f28-destructive job?

Reply:
Right, FYi, per the commit - 2fa7e4ab694c, seem like some team added the setting in rotate way. If they need, then can file other PR to add it later.